vaas/vaas/resources/dev-data.yaml

```diff
@@ -147,12 +59,290 @@
 - model: router.redirect
   pk: 1
   fields:
-    src_domain: 1
+    src_domain: 3
     condition: req.url ~ "/source_url"
     destination: http://mydomain.com/destination_url
     action: 301
     priority: 250
     preserve_query_params: false
+- model: router.redirect
```

Comment:
To be sure, should we add the same number of redirects for the development environment?

Reply:
Removed the extra redirects in [9630c5a](https://github.com/allegro/vaas/pull/518/commits/9630c5af87d89da4039a9cc770184caf3731c065)